src/Quiz/Quiz.js

```diff
@@ -74,12 +95,7 @@ const Quiz = (props) => {
           onChange={handleChange}
         />
       </div>
-      <div className="trip-gender-field">
-        <h2>2. I'm packing for:</h2>
-        <button name='female' onClick={(event) => handleClick(event, 'gender')}>Female</button>
-        <button name='male' onClick={(event) => handleClick(event, 'gender')}>Male</button>
-        <button name='non-binary' onClick={(event) => handleClick(event, 'gender')}>Non-Binary</button>
-      </div>
+
       <div className="trip-location-field">
         <h2>3. Going to:</h2>
```

Comment:
We're going to want to make a new component for inputs to tidy up Quiz and get the questions mapped in order.